pyiceberg/table/inspect.py

```diff
@@ -95,7 +139,21 @@ def snapshots(self) -> "pa.Table":
         )
 
     def entries(self, snapshot_id: Optional[int] = None) -> "pa.Table":
-        import pyarrow as pa
+        """ Generate a table of manifest entries for a specific snapshot.
+
+         This method retrieves metadata for all manifest entries within a given snapshot, 
+         including file-level statistics such as column sizes and value counts.
+
+         Args:
+            snapshot_id (Optional[int]): The ID of the snapshot to retrieve entries for. 
+              If None, entries for the current snapshot are returned.
```

Comment:
i like this description of how `snapshot_id` is used, can we apply this for all similar functions
perhaps something more generic
```suggestion
         Args:
            snapshot_id (Optional[int]): The ID of the snapshot to retrieve. If None, the current snapshot is used.
```

Reply:
Thank you for the feedback! I’m glad you liked the description. I’ll review other similar functions and update their docstrings to use a more generic and consistent phrasing as suggested. Let me know if there are any additional improvements you'd like to see